plugin-admin/generate_plugins.js

```diff
@@ -0,0 +1,101 @@
+// Create empty object structure
```

Comment:
this comment is much to low level - these operations are evident in the code itself

instead, note in one or two sentences, what the *purpose* of the script is